core/src/main/java/com/amplifyframework/datastore/ErrorType.kt

```diff
@@ -0,0 +1,5 @@
+package com.amplifyframework.datastore
+
+ enum class ErrorType {
```

Comment:
Is this needed?  I don't think it's used anymore

Reply:
Done